jragard/kenzie-testing
# Fizzbuzz

# Changes Made
- Proposed alteration of the assessment from styling DOM elements to the traditional FizzBuzz
- Broke assessment into 3 functions (fizzbuzz, writeToDOM, and fizzbuzzPrime [Bonus])
- Implemented tests for fizzbuzz and fizzbuzzPrime function
- Updated test file to fit proposed structure
- Included run script

# How to Test this PR
- pull down the fizzbuzz branch with `git fetch`
- run `git checkout fizzbuzz`
- cd 'Fizzbuzz`
- run `mocha` or `npm test`
    - Check to see all tests passing
- Check the tests themselves as well as the code they are testing to make sure we are testing what we think we are testing
- Check the recommendations to instructions and make sure they are clear and will direct students down the path we are intending to test


# Reviewing
 - If all looks good, merge `fizzbuzz` into `dev` and then delete `fizzbuzz` to keep our repo clean of old branches
- If all does not look good, either make changes or request changes through the PR notes on github
